### ISSUE TITLE
Template(Jackett) unable to Reinstall after Removal

### ISSUE THREAD

**Issue by porter9992** [BiglySoftware/BiglyBT#3439]
Hi Porgster,
I had this case open that you advised me on (https://github.com/BiglySoftware/BiglyBT/issues/3105), 
I hadn't used BiglyBT since earlier this year, so I updated to 3.7 from the old version, and I was working on re-attaching to Jackett (had to restore from hard drive corruption) and I had deleted the Jackett Search Template by accident.  

I am trying to re-add the Jackett template and I do get the dialog box that Jackett already exists (per other case comment), so I incremented the ID# up and tried a few more increments but nothing happens when I import JSON from clipboard.  No dialog box and no new entries in the search templates. 

I have already opened & closed a few times, but no change.  (Win 10 Pro)

**Comment by parg** [comment]
Seems you have to change the "name" as well, as the id, try "Jackett2" or something.

**Comment by Havokdan** [comment]
You have to remove the jackett in the templates page as well as in the rss section.

**Comment by porter9992** [comment]
@parg @Havokdan , renaming did let it be added.  
I wasn't sure where the templates page was, except for the "Add/Edit Templates" that is inside the search results.  But when I click it I've kept getting the error attached.
![Template Page](https://github.com/user-attachments/assets/11702bbc-f3a7-4193-80c6-8d7872118914)

**Comment by parg** [comment]
right-click on the search icon (to the left of the search bar at the top of BiglyBT) to get a menu -> Templates

**Comment by Havokdan** [comment]
![Sem título](https://github.com/user-attachments/assets/af97622f-a31d-4673-a240-52f32f046f59)
Here too

**Comment by porter9992** [comment]
> right-click on the search icon (to the left of the search bar at the top of BiglyBT) to get a menu -> Templates

Right, that's the place I've been doing the importing. See pic, which I have the new renamed "Jcktt".  This is the place I had removed the "Jackett" entry and results where we are now. :)
![Search Properties](https://github.com/user-attachments/assets/da0b4526-af36-4cd5-bc08-5991d475213c)

@Havokdan , I don't see how to get the templates selection under that extra section or another? See this pic.
![Uploading Bigly Interface.jpg…]()

**Comment by parg** [comment]
So it is working? If so please close the issue :)

**Comment by porter9992** [comment]
Yes, just the "Jackett" name can never be used again.
